### ISSUE TITLE
ValueError: Unknown encoder_type UNI

### ISSUE THREAD

**Issue by siddas27** [DongjunLee/dmn-tensorflow#2]
```
File "/home/sid/gitrepo/dmn-tensorflow-1/dynamic_memory/encoder.py", line 57, in build
    raise ValueError(f"Unknown encoder_type {self.encoder_type}")
ValueError: Unknown encoder_type UNI
```

**Comment by ashwinsk** [comment]
i think internally he is referencing it as in without caps; so it would be "uni" and "bi"
check this out https://github.com/DongjunLee/dmn-tensorflow/blob/master/dynamic_memory/encoder.py

**Comment by DongjunLee** [comment]
Fixed by pull request #3
